### PR TITLE
tag rm cmd

### PR DESCRIPTION
**Pull request by ongspxm**

### CODE REVIEW

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -279,6 +279,7 @@ public boolean equals(Object obj) {
 
         // state check
         ModelManager other = (ModelManager) obj;
+        System.out.println(filteredEntries.equals(other.filteredEntries));
```
Comment (jhengy) [inline src/main/java/seedu/address/model/ModelManager.java]: Shall we remove this?

File: src/main/java/seedu/address/logic/commands/TagRmCommand.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+
+import seedu.address.model.Model;
+import seedu.address.model.entry.ResumeEntry;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Add tags to specified entry.
```
Comment (anubh-v) [inline src/main/java/seedu/address/logic/commands/TagRmCommand.java]: I think you mean remove tags from entry?

File: src/main/java/seedu/address/logic/commands/TagRmCommand.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            + "Parameters: INDEX "
+            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_TAG + "java ";
+
+    private final Index index;
+    private Set<Tag> tags;
+
+    /**
+     * Creates an TagRmCommand to add the specified {@code ResumeEntry}
```
Comment (anubh-v) [inline src/main/java/seedu/address/logic/commands/TagRmCommand.java]: Should it be to `edit the specified ResumeEntry` ?

File: src/main/java/seedu/address/logic/parser/TagRmCommandParser.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+import java.util.Set;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.TagRmCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Parses input arguments and creates a new AddBulletCommand object
```
Comment (anubh-v) [inline src/main/java/seedu/address/logic/parser/TagRmCommandParser.java]: Remember to update the header comments

File: src/main/java/seedu/address/logic/commands/TagRmCommand.java [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+        model.updateEntry(entryToEdit, editedEntry);
+        model.updateFilteredEntryList(Model.PREDICATE_SHOW_ALL_ENTRIES);
+        model.commitAddressBook();
+
+        return new CommandResult(String.format(MESSAGE_SUCCESS));
+    }
+
+    /**
+     * Takes an entry and return an {@code ResumeEntry} with updated details specified
```
Comment (anubh-v) [inline src/main/java/seedu/address/logic/commands/TagRmCommand.java]: Javadoc comment could be more detailed I guess?

**Comment by ongspxm** [comment]
@anubh-v @jhengy good catch guys, ready for second round

**Comment by anubh-v** [comment]
Looks good; can help resolve the conflict?

**Comment by anubh-v** [comment]
I resolved the conflict; can check if it's ok?

**Comment by ongspxm** [comment]
:+1: once green can merge

**Pull request merged**